spec/models/user_auth_token_spec.rb

```diff
@@ -5,33 +5,54 @@
 RSpec.describe UserAuthToken do
   fab!(:user)
 
-  it "can remove old expired tokens" do
-    SiteSetting.verbose_auth_token_logging = true
+  describe ".cleanup!" do
+    it "can remove old expired tokens" do
+      freeze_time Time.zone.now
+      SiteSetting.maximum_session_age = 1
 
-    freeze_time Time.zone.now
-    SiteSetting.maximum_session_age = 1
+      token =
+        UserAuthToken.generate!(
+          user_id: user.id,
+          user_agent: "some user agent 2",
+          client_ip: "1.1.2.3",
+        )
 
-    token =
-      UserAuthToken.generate!(
-        user_id: user.id,
-        user_agent: "some user agent 2",
-        client_ip: "1.1.2.3",
-      )
+      freeze_time 1.hour.from_now
+      UserAuthToken.cleanup!
 
-    freeze_time 1.hour.from_now
-    UserAuthToken.cleanup!
+      expect(UserAuthToken.where(id: token.id).count).to eq(1)
 
-    expect(UserAuthToken.where(id: token.id).count).to eq(1)
+      freeze_time 1.second.from_now
+      UserAuthToken.cleanup!
 
-    freeze_time 1.second.from_now
-    UserAuthToken.cleanup!
+      expect(UserAuthToken.where(id: token.id).count).to eq(1)
 
-    expect(UserAuthToken.where(id: token.id).count).to eq(1)
+      freeze_time UserAuthToken::ROTATE_TIME.from_now
+      UserAuthToken.cleanup!
 
-    freeze_time UserAuthToken::ROTATE_TIME.from_now
-    UserAuthToken.cleanup!
+      expect(UserAuthToken.where(id: token.id).count).to eq(0)
+    end
 
-    expect(UserAuthToken.where(id: token.id).count).to eq(0)
+    it "deletes old logs excluding certain types" do
```

Comment:
Might be nice to spell out here which types are excluded. It's in the test examples below, but useful to have in the test description IMO.